docs/README.md

```diff
@@ -34,18 +34,3 @@ This directory contains all of the documentation on contributing to freeCodeCamp
 
 <a href="/docs/how-to-work-on-guide-articles.md">1. Style guide for creating guide articles.</a><br>
 <a href="/docs/how-to-work-on-coding-challenges.md">2. Style guide for creating coding challenges.</a>
-
```

Comment:
What do you plan to do with lines 37-51 which you removed from the README.md?

Reply:
They were removed mistakenly, re-added